src/i18n/en.json

```diff
@@ -33,9 +33,10 @@
   },
   "reports_title": "Doppler | Reports",
   "reports_box": {
-    "visits": "Total Visits",
+    "visits": "Visits with emails",
```

Comment:
What do you think about renaming this key?

Reply:
i asumed this has with emails, but i can change 😃